### ISSUE TITLE
FR: show recipient's e-mail address

### ISSUE THREAD

**Issue by YZgitter** [roundcube/roundcubemail#9219]
I want to see the **actual full e-mail address** of the recipient and not the stupid "user friendly name" or whatever it's called.
And I don't want to hover over with a mouse to see it, it should be displayed there all the time.

Please, for the love of god, add a setting for this.

**Comment by alecpl** [comment]
There is an option for this, but it does not apply to the messages list. The messages list is not always wide enough to display both name and address, so I suppose it would have to display only the address.

**Comment by stef204** [comment]
this feature would be a very welcome addition.

> Seeing the actual address there can prevent important mistakes

**Comment by LevitatingBusinessMan** [comment]
I agree that this feature is missing, which is why I made the userscript linked to by @stef204.

My contacts often have multiple addresses and without this little hack I am continuously paranoid of sending to the wrong email.
